metricbeat/module/docker/diskio/diskio.go

```diff
@@ -35,22 +35,44 @@ func init() {
 	)
 }
 
+// Config "imports" the base module-level config, plus our metricset options
+type Config struct {
+	TLS       *docker.TLSConfig `config:"ssl"`
+	DeDot     bool              `config:"labels.dedot"`
+	SkipMajor []uint64          `config:"skip_major"`
+}
+
+var defaultMajorDev = []uint64{9, 253}
```

Comment:
Can you document what these default values mean and why we need this? Maybe just link to the issue for an explanation.

Reply:
The `9` and `253` would be clearer as named constants.